### PR TITLE
Povio - command-score should be robust to duplicated letters.

### PR DESCRIPTION
**Pull request by JasaNiklanovic**
Fixes https://github.com/superhuman/superhuman/issues/7428

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -106,7 +106,6 @@ function commandScoreInner(string, abbreviation, lowerString, lowerAbbreviation,
         }
 
         if (score < SCORE_TRANSPOSITION &&
-                lowerString.charAt(index - 1) === lowerAbbreviation.charAt(abbreviationIndex + 1) &&
```
Comment (JasaNiklanovic) [inline index.js:109]: Tadl will match talent. This shouldn't happen. Lets add a condition here.

Comment (JasaNiklanovic) [reply index.js:109]: @ConradIrwin i added a test that "tadl" does not match "talent". If you have any other concerns here let me know.

Comment (ConradIrwin) [reply index.js:109]: @JasaNiklanovic A failing test is that `tadlent` should not match `talent`.

Comment (JasaNiklanovic) [reply index.js:109]: Look at insertion instead of transposition.

**Comment by JasaNiklanovic** [comment]
@ConradIrwin i think i figured this out, before i had base string and comparison string mixed up. Adding another optional condition to score transposition (allowing duplicate consecutive letters) seems to do the trick.

**Pull request merged**